charts/datadog/templates/NOTES.txt

```diff
@@ -355,6 +355,18 @@ You are using datadog.kubeStateMetricsCore.enabled but you disabled the cluster
 To enable it please set clusterAgent.enabled to 'true'.
 {{- end }}
 
+{{- if and .Values.datadog.kubeStateMetricsCore.useClusterCheckRunners (not .Values.clusterChecksRunner.enabled)}}
+
+###################################################################################
+####               WARNING: Suboptimal Cluster Checks Runner configuration     ####
+###################################################################################
+
+You have `datadog.kubeStateMetricsCore.useClusterCheckRunners` enabled and `clusterChecksRunner.enabled` disabled.
```

Comment:
```suggestion
The option `datadog.kubeStateMetricsCore.useClusterCheckRunners` is enabled and `clusterChecksRunner.enabled` disabled.
```

Reply:
I think we don't directly address the user in the doc, I could be wrong

Reply:
We are not consistent across all notes but quite a few directly address the user ([link](https://cs.github.com/DataDog/helm-charts/blob/b112e1e3f37cd0fd779701d8f6030d6f9279fd5e/charts/datadog/templates/NOTES.txt?q=org%3ADataDog+path%3ANOTES.txt+repo%3Ahelm-charts+you#L61)) so will keep it as is.